colorbleed/plugins/maya/publish/increment_current_file_deadline.py

```diff
@@ -0,0 +1,75 @@
+import pyblish.api
+
+
+class IncrementCurrentFileDeadline(pyblish.api.ContextPlugin):
+    """Submit available render layers to Deadline
+
+    Renders are submitted to a Deadline Web Service as
+    supplied via the environment variable AVALON_DEADLINE
+
+    """
+
+    label = "Increment current file"
+    order = pyblish.api.IntegratorOrder + 9.0
+    hosts = ["maya"]
+    families = ["colorbleed.renderlayer"]
+
+    def process(self, context):
+
+        from maya import cmds
+
+        from colorbleed.action import get_errored_plugins_from_data
+
+        plugins = get_errored_plugins_from_data(context)
+
+        if any(plugin.__name__ == "MindbenderSubmitDeadline"
```

Comment:
Did you test whether this works? Maybe once force the `submit_to_deadline` plug-in to raise an error and see whether this is picked up here.